src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.Generated.cs

```diff
@@ -234,7 +234,7 @@ internal sealed class ActionCallInstruction : CallInstruction
 
         public ActionCallInstruction(MethodInfo target)
         {
-            _target = (Action)target.CreateDelegate(typeof(Action));
+            _target = target.CreateDelegate<Action>();
```

Comment:
It may be good idea to rollback the changes in this file as well. The types in this file are instantiated many times and the extra generic method instantiations will add to the footprint.

Reply:
Or maybe revert all uses under System.Linq.Expressions/src for consistency.

Reply:
I hadn't thought about the potential problems that having mixed API's can potentially bring. Let me revert all of System.Linq.Expressions then.